wormchain/build/config/config.toml

```diff
@@ -93,7 +93,7 @@ laddr = "tcp://0.0.0.0:26657"
 # A list of origins a cross-domain request can be executed from
 # Default value '[]' disables cors support
 # Use '["*"]' to allow any origin
-cors_allowed_origins = ["*", ]
+cors_allowed_origins = []
```

Comment:
this may break things

Reply:
there are no websites that interact with the local development build.